scripts/count_opcodes.py

```diff
@@ -139,6 +141,7 @@ def report(source, filename, verbose, bias):
     for co in all_code_objects(code):
         cache_counter = CacheCounter(counter)
         counter[NCODEOBJS] += 1
+        counter[CO_CONSTS_SIZE] += sys.getsizeof(co.co_consts)
```

Comment:
I never understand exactly what sys.getsizeof() counts. Why not count the number of items in the tuple?

Reply:
That gives 48,822 --> 46,682, about 4.3%.

Reply:
Well it turns out` sys.getsizeof(t)` for a tuple t returns `40 + 8*len(t)`. Since every code object has a `co_consts` field, we could compute the getsizeof-based number as 8 times the count-based number plus 40 times the number of code objects, right?

I think reporting it as "we have 4.3% fewer constants" makes more sense than reporting "we have 2.3% less memory tied up in co_consts tuples". Because the constants are just a small fraction of the overall memory use for code objects.